### DISCUSSION TITLE
How can I get rid of default desktop entries?

### DISCUSSION THREAD

**Discussion by floreapaun** [lxqt/lxqt#2657] [Q&A] [answered]
I've just installed Lubuntu with LXQt. 
I have four desktop entries that I cannot delete. 
If I delete them and reboot, they reappear: Computer, User, Lubuntu Manual, and Network. 
How can I permanently remove them?

**Comment by isf63** [comment, marked as answer]
Right click Desktop → Desktop Preferences → Advanced, and toggle the shortcuts.
